README.md

```diff
@@ -97,6 +97,24 @@ BRANCHOUT_NAME=shorter-name
 
 to the Branchoutfile and ideally use the same value as the optionalDirName during branchout init.
 
+### Personal customisation
+
+Sometimes the branchout defaults don't work for a particular scenario even though they do work for most of your colleagues.
+One example is that the `${HOME}` places the two root directories on a very slow network drive.
+Another might be that you already have a tree with your code and you want to keep it all together.
+In cases like these you can use the following two files to ????
+
+```
+~/.config/Branchoutrc?
+${BRANCHOUT_BASE}/.branchoutrc
```

Comment:
whats branchout base?

Reply:
are you thinking a common place like the metarepo? i guess that would work

BRANCHOUT_PROJECTION_DIRECTORY

Reply:
i guess it could be absolute but that just is hard if you have multiple platforms

Reply:
Good question, both of these two lines were for you to confirm/rewrite in a way that makes sense. They were just guesses.

I'll add a glossary from the source code, maybe in a different file? And we can get on the same page terminology/jargon wise there.